### PR TITLE
Create a new monitoring enterprise doc

### PR DESCRIPTION
**Pull request by mluu12**
Closes https://github.com/influxdata/docs-v2/issues/3077

Uses community templates to tell the users how to monitor enterprise with a template. 

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-enterprise.md [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
   enterprise_influxdb_1_9:
-    name: Monitor InfluxDB
+    name: Monitor Enterprise
     weight: 80
-    parent: Administration
+    parent: Monitor InfluxDB
 ---
 
-**On this page**
+Monitoring is the act of proactively observing changes in data throughout time. There are multiple ways to monitor InfluxDB Enterprise.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-enterprise.md:15]:
```suggestion
Monitoring is the act of proactively observing changes in data over time. There are multiple ways to monitor InfluxDB Enterprise. 
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-enterprise.md [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+    name: Monitor Enterprise
     weight: 80
-    parent: Administration
+    parent: Monitor InfluxDB
 ---
 
-**On this page**
+Monitoring is the act of proactively observing changes in data throughout time. There are multiple ways to monitor InfluxDB Enterprise. 
+
+If you want to monitor Enterprise, do one of the following:
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-enterprise.md:17]:
```suggestion
If you want to monitor an Enterprise cluster, do one of the following: 
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+8. (Optional) [Monitor with InfluxDB Insights and Aware](#monitor-with-influxdb-insights-and-aware)
+
+## Review requirements
+
+Before you begin, make sure you have access to the following:
+
+ - InfluxDB Cloud account ([sign up for free here](https://cloud2.influxdata.com/signup))
+ - Command line access to a machine [running InfluxDB Enterprise 1.x](/enterprise_influxdb/v1.9/introduction/install-and-deploy/) and permissions to install Telegraf on this machine
+ - Internet connectivity from the machine running InfluxDB Enterprise 1.x and Telegraf to InfluxDB Cloud
+ - Sufficient resource availability to install the template. InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits)
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md:33]:
```suggestion
 - An InfluxDB Cloud account. ([Sign up for free here](https://cloud2.influxdata.com/signup)).
 - Command line access to a machine [running InfluxDB Enterprise 1.x](/enterprise_influxdb/v1.9/introduction/install-and-deploy/) and permissions to install Telegraf on this machine.
 - Internet connectivity from the machine running InfluxDB Enterprise 1.x and Telegraf to InfluxDB Cloud.
 - Sufficient resource availability to install the template. (InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits).)
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+## Install the InfluxDB Enterprise Monitoring template
+
+The InfluxDB Enterprise Monitoring template includes a Telegraf configuration that sends InfluxDB Enterprise metrics to an InfluxDB endpoint and a dashboard that visualizes the metrics.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md:37]:
```suggestion
The InfluxDB Enterprise Monitoring template includes a Telegraf configuration that sends InfluxDB Enterprise metrics to an InfluxDB endpoint, and a dashboard that visualizes the metrics.
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+   
+    {{< keep-url >}}
+     ```toml
+     [[inputs.prometheus]]
+       urls = ["http://localhost:8086/metrics"]
+       username = "$INFLUX_USER"
+       password = "$INFLUX_PASSWORD"
+     ``` 
+     
+     If you're using unique URLs or have security set up for your `/metrics` endpoint, configure those options here and save the updated configuration.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md:95]:
```suggestion
     If you're using unique URLs or have authentication set up for your `/metrics` endpoint, configure those options here and save the updated configuration.
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+Set up Telegraf to scrape metrics from InfluxDB Enterprise to send to your InfluxDB Cloud account.
+
+On each InfluxDB Enterprise instance you want to monitor, do the following:
+
+1. Go to **Load Data > Telegraf** [in your InfluxDB Cloud account](https://cloud2.influxdata.com/).
+2. Click **Setup Instructions** under **monitoring-enterprise-1.x**. 
+3. Complete the Telegraf Setup instructions. If you are using environment variables, set them up now. 
+
+      {{% note %}}
+For your API token, generate a new token or use an existing All Access token. If you run Telegraf as a service, edit your init script to set the environment variable and ensure its available to the service.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md:111]:
```suggestion
For your API token, generate a new token or use an existing All Access token. If you run Telegraf as a service, edit your init script to set the environment variable and ensure that it's available to the service.
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+4. Set the amount of time to wait before switching to a critical alert.
+5. Save the Check and click on **View History** of the Check under the gear icon to verify it is running.  
+
+## Create a notification endpoint and rule
+
+To receive a notification message when your deadman check is triggered, create a [notification endpoint](#create-a-notification-endpoint) and [rule](#create-a-notification-rule). 
+
+### Create a notification endpoint 
+
+InfluxData supports different endpoints: Slack, PagerDuty, and HTTP. Slack is free for all users, while PagerDuty and HTTP are exclusive to the Usage-Based Plan.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md:160]:
```suggestion
InfluxDB Cloud supports different endpoints: Slack, PagerDuty, and HTTP. Slack is free for all users, while PagerDuty and HTTP are exclusive to the Usage-Based Plan. 
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/overview.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+title: Overview 
+description: Learn about the different ways to monitor InfluxDB Enterprise. 
+menu:
+  enterprise_influxdb_1_9:
+    name: Overview
+    weight: 70
+    parent: Monitor InfluxDB
+---
+
+## Monitoring and looking at data
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/overview.md:11]: Maybe?
```suggestion
## Monitoring and viewing internal metrics
```

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-oss.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
+---
+title: Monitor InfluxDB Enterprise using OSS
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-oss.md:2]: This doc needs further work to change Cloud-related steps to OSS 2.0.

File: content/influxdb/cloud/influxdb-templates/monitor-enterprise.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
+---
```
Comment (pierwill) [inline content/influxdb/cloud/influxdb-templates/monitor-enterprise.md:1]: Is this a duplicate of the document above: https://github.com/influxdata/docs-v2/pull/3078/files#diff-880efcbec26052e4ff2a01746f277ef0dbdebbf904b87c46054051715f094d16?

We'll likely just want to link readers to that guide.

**Review by pierwill — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-oss.md (hunk 9 above)
Comment (mluu12) [reply content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-oss.md:2]: Finished the changes!

File: content/influxdb/cloud/influxdb-templates/monitor-enterprise.md (hunk 10 above)
Comment (mluu12) [reply content/influxdb/cloud/influxdb-templates/monitor-enterprise.md:1]: I also believe that I adjusted this document so that they're separate!

**Pull request merged**